pkg/store/tsdb.go

```diff
@@ -63,21 +73,68 @@ type ReadWriteTSDBStore struct {
 
 // NewTSDBStore creates a new TSDBStore.
 // NOTE: Given lset has to be sorted.
-func NewTSDBStore(logger log.Logger, db TSDBReader, component component.StoreAPI, extLset labels.Labels) *TSDBStore {
+func NewTSDBStore(logger log.Logger, db TSDBReader, component component.StoreAPI, extLset labels.Labels, metricNameFilterEnabled bool) *TSDBStore {
 	if logger == nil {
 		logger = log.NewNopLogger()
 	}
-	return &TSDBStore{
+
+	var (
+		storeFilter       filter.StoreFilter
+		startFilterUpdate bool
+	)
+
+	storeFilter = filter.AllowAllStoreFilter{}
+	if metricNameFilterEnabled {
+		startFilterUpdate = true
+		storeFilter = filter.NewCuckooMetricNameStoreFilter(1000000) // about 1MB on 64bit machines.
```

Comment:
we can extract this number into a constant like
```
const megabyte = 1024 *1024
```

Reply:
actually we set just capacity (which I now moved to constant), comment about memory comes from package itself: https://github.com/seiflotfy/cuckoofilter/blob/master/cuckoofilter.go#L18-L20. Now I have deleted comment at all :)